pkg/gameserverallocations/controller_test.go

```diff
@@ -75,7 +75,7 @@ func TestControllerAllocationHandler(t *testing.T) {
 			return true, gs, nil
 		})
 
-		stop, cancel := agtesting.StartInformers(m)
+		stop, cancel := agtesting.StartInformers(m, c.gameServerSynced)
```

Comment:
if the controller has the WaitForCacheSync, is it really needed to call StartInformers that does the same thing (e.g. calling [WaitForCacheSync](https://github.com/GoogleCloudPlatform/agones/blob/master/pkg/testing/controller.go#L73))

Reply:
The `WaitforcacheSync` is called in the controller `.Run()` command - so just instantiating the controller doesn't fire the `WaitforcacheSync`.

And in tests (I've found anyway), you often don't want everything that happens inside the `Run()` command to fire.

This was following the  patterns of the [sample controller](https://github.com/kubernetes/sample-controller/blob/master/controller.go#L159)